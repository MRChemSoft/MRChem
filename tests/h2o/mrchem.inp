# vim:syntax=sh:

rel_prec = 1.0e-3                 # Overall relative precision

MRA {
  order = 6                       # Polynomial order
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
$coords
O       0.0000  0.0000  -0.1250
H      -1.4375  0.0000   1.0250
H       1.4375  0.0000   1.0250
$end
}

WaveFunction {
  method = DFT                    # Wave function method (HF or DFT)
}

DFT {
  exact_exchange = 0.25
$functionals
PBEX    0.75
PBEC    1.00
$end
}

Properties {
  scf_energy = true               # Compute ground state energy
  dipole_moment = true            # Compute electrostatic dipole moment
}

SCF {
  orbital_thrs = 1.0e-2           # Convergence threshold in orbital residual
  initial_guess = GTO             # Type of initial guess: none, mw, gto
  kain = 3                        # Length of KAIN iterative history
  rotation = 5                    # Diagonalize Fock matrix every n'th iteration
  canonical = false               # Use canonical of localized orbitals
}

