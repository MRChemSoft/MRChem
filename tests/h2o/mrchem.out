


************************************************************
***                                                      ***
***    MRChem 1.0.0 (rev. e6faeac)                       ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Jonas Juselius   <jonas.juselius@uit.no>          ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***                                                      ***
************************************************************


Print level  : 0

BLAS was NOT found, Eigen will be used instead!

+++ Parallel execution: 
  MPI hosts available     : 1
  Threads/host            : 16
  Total used CPUs         : 16



============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 6
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 32.00000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [   -32.00000   -32.00000   -32.00000 ]
 upper bounds     = [    32.00000    32.00000    32.00000 ]
 total length     = [    64.00000    64.00000    64.00000 ]
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     O            8.00000e+00   1.00000e-03   5.10129e-04
  2     H            1.00000e+00   1.00000e-03   1.63241e-02
  3     H            1.00000e+00   1.00000e-03   1.63241e-02
============================================================


============================================================
        Setting up occupied orbitals (closed-shell)
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    u    2                       1.000000407906976e+00
    1    u    2                       1.000000130473305e+00
    2    u    2                       9.999999983977805e-01
    3    u    2                       1.000000017739475e+00
    4    u    2                       9.999999973891708e-01
------------------------------------------------------------
                 Wall time: 4.64469e-01 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-04
------------------------------------------------------------
 Nuclear potential                    1256      7.25660e-02
 Coulomb density                       680      1.13512e-01
 Coulomb potential                     328      3.31407e-01
 Hartree-Fock exchange                 488      6.33914e+00
 XC density                            680      1.05034e-01
 XC density gradient                  2040      1.37160e-02
 XC preprocess xcfun                  1360      6.89220e-02
 XC evaluate xcfun                    2040      1.53723e-01
 XC energy                             680      4.40000e-05
 XC potential                          680      9.14070e-02
------------------------------------------------------------
                 Wall time: 7.29565e+00 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                    Localizing orbitals
------------------------------------------------------------
 Computing Foster-Boys matrix                   7.74217e-01
 Converged after iteration                               24
 Rotating orbitals                              7.63120e-02
------------------------------------------------------------
                 Wall time: 8.50602e-01 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       9.039235444235468e+00
 Electronic energy                   -8.537785165398606e+01
------------------------------------------------------------
 Total energy                        -7.633861620975058e+01
------------------------------------------------------------
                 Wall time: 2.87260e-01 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -1.90836e+01
 Creating operator with lambda                 -6.86959e-01
 Re-using operator with lambda                 -6.86959e-01
 Creating operator with lambda                 -4.87999e-01
 Re-using operator with lambda                 -4.87999e-01
------------------------------------------------------------
                 Wall time: 3.03875e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.99742e-01  0.00000e+00   456     0  2.6e-04  5.2e-01
  1  9.98814e-01  0.00000e+00   360     0  1.2e-03  4.5e-01
  2  9.98814e-01  0.00000e+00   360     0  1.2e-03  4.4e-01
  3  1.00650e+00  0.00000e+00   392     0  6.5e-03  4.8e-01
  4  1.00650e+00  0.00000e+00   392     0  6.5e-03  4.8e-01
------------------------------------------------------------
                 Wall time: 2.36680e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.10000e-05
 Copy orbitals                                  1.63650e-02
 Copy orbital updates                           2.23010e-02
------------------------------------------------------------
                 Wall time: 3.87640e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     872      5.07290e-02
 Coulomb density                       616      9.75690e-02
 Coulomb potential                     136      1.59497e-01
 Hartree-Fock exchange                 616      3.68027e+00
 XC density                            616      1.00009e-01
 XC density gradient                  1848      1.35390e-02
 XC preprocess xcfun                  1232      6.32580e-02
 XC evaluate xcfun                    1848      1.47643e-01
 XC energy                             616      3.60000e-05
 XC potential                          616      8.61750e-02
------------------------------------------------------------
                 Wall time: 4.40391e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -1.91243e+01   4.64417e-03        552    u    2    1
  1  -7.11458e-01   2.70435e-02        488    u    2    0
  2  -7.11067e-01   2.70435e-02        488    u    2    0
  3  -5.20882e-01   5.90718e-02        456    u    2    0
  4  -5.20873e-01   5.90718e-02        456    u    2    0
------------------------------------------------------------
 Total error:                            9.19958e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       7.584575687411638e+01     7.58458e+01    1
 N-E          -1.987270186359020e+02    -1.98727e+02    1
 Coulomb       4.676190203152783e+01     4.67619e+01    1
 Exchange     -2.235018318017563e+00    -2.23502e+00    1
 X-C          -7.023473605710731e+00    -7.02347e+00    1
------------------------------------------------------------
 Electronic   -8.537785165398606e+01    -8.53779e+01    1
 Nuclear       9.039235444235468e+00     9.03924e+00    1
------------------------------------------------------------
 Total        -7.633861620975058e+01    -7.63386e+01    1
============================================================


################ Wall time: 9.54879e+00 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                    Localizing orbitals
------------------------------------------------------------
 Computing Foster-Boys matrix                   9.22787e-01
 Converged after iteration                                2
 Rotating orbitals                              6.09360e-02
------------------------------------------------------------
                 Wall time: 9.83794e-01 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       9.039235444235468e+00
 Electronic energy                   -8.542489449633504e+01
------------------------------------------------------------
 Total energy                        -7.638565905209957e+01
------------------------------------------------------------
                 Wall time: 2.73922e-01 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -1.91187e+01
 Creating operator with lambda                 -7.10000e-01
 Creating operator with lambda                 -7.09607e-01
 Creating operator with lambda                 -5.25134e-01
 Re-using operator with lambda                 -5.25134e-01
------------------------------------------------------------
                 Wall time: 4.13439e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.99219e-01  0.00000e+00   456     0  7.8e-04  5.0e-01
  1  9.95406e-01  0.00000e+00   360     0  4.6e-03  4.4e-01
  2  9.95550e-01  0.00000e+00   360     0  4.5e-03  4.4e-01
  3  9.97393e-01  0.00000e+00   392     0  2.6e-03  4.7e-01
  4  9.97397e-01  0.00000e+00   392     0  2.6e-03  4.7e-01
------------------------------------------------------------
                 Wall time: 2.31391e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             8.00000e-06
 Copy orbitals                                  1.60160e-02
 Copy orbital updates                           1.99360e-02
------------------------------------------------------------
                 Wall time: 3.60400e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     872      5.05590e-02
 Coulomb density                       616      9.68210e-02
 Coulomb potential                     136      1.57900e-01
 Hartree-Fock exchange                 616      3.64789e+00
 XC density                            616      9.79460e-02
 XC density gradient                  1848      1.36080e-02
 XC preprocess xcfun                  1232      6.27690e-02
 XC evaluate xcfun                    1848      1.46881e-01
 XC energy                             616      4.00000e-05
 XC potential                          616      8.59010e-02
------------------------------------------------------------
                 Wall time: 4.36597e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -1.90955e+01   7.06440e-03        552    u    2    1
  1  -7.04796e-01   1.76531e-02        488    u    2    0
  2  -7.04790e-01   1.76341e-02        488    u    2    0
  3  -5.15574e-01   1.55848e-02        456    u    2    0
  4  -5.15582e-01   1.55838e-02        456    u    2    0
------------------------------------------------------------
 Total error:                            3.40329e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       7.584064102026434e+01    -5.11585e-03    1
 N-E          -1.985240867184581e+02     2.02932e-01    1
 Coulomb       4.648331286963693e+01    -2.78589e-01    1
 Exchange     -2.225811001506089e+00     9.20732e-03    1
 X-C          -6.998950666272204e+00     2.45229e-02    1
------------------------------------------------------------
 Electronic   -8.542489449633504e+01    -4.70428e-02    1
 Nuclear       9.039235444235468e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.638565905209957e+01    -4.70428e-02    1
============================================================


################ Wall time: 9.69463e+00 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       9.039235444235468e+00
 Electronic energy                   -8.545704668251783e+01
------------------------------------------------------------
 Total energy                        -7.641781123828235e+01
------------------------------------------------------------
                 Wall time: 2.77452e-01 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -1.90955e+01
 Creating operator with lambda                 -7.04796e-01
 Re-using operator with lambda                 -7.04796e-01
 Creating operator with lambda                 -5.15574e-01
 Re-using operator with lambda                 -5.15574e-01
------------------------------------------------------------
                 Wall time: 3.02910e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.99528e-01  0.00000e+00   456     0  4.7e-04  5.1e-01
  1  9.98347e-01  0.00000e+00   360     0  1.7e-03  4.4e-01
  2  9.98304e-01  0.00000e+00   360     0  1.7e-03  4.4e-01
  3  1.00242e+00  0.00000e+00   392     0  2.4e-03  4.8e-01
  4  1.00242e+00  0.00000e+00   392     0  2.4e-03  4.8e-01
------------------------------------------------------------
                 Wall time: 2.35024e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             8.00000e-06
 Setup linear system                            5.67990e-02
 Solve linear system                            1.20000e-05
 Expand solution                                6.49130e-02
------------------------------------------------------------
                 Wall time: 1.21820e-01 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     872      5.05230e-02
 Coulomb density                       616      9.77930e-02
 Coulomb potential                     136      1.58573e-01
 Hartree-Fock exchange                 616      3.68985e+00
 XC density                            616      9.79220e-02
 XC density gradient                  1848      1.37970e-02
 XC preprocess xcfun                  1232      6.22560e-02
 XC evaluate xcfun                    1848      1.47506e-01
 XC energy                             616      3.70000e-05
 XC potential                          616      8.06250e-02
------------------------------------------------------------
                 Wall time: 4.40457e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -1.91095e+01   3.16433e-03        552    u    2    1
  1  -7.09513e-01   5.72975e-03        488    u    2    1
  2  -7.09551e-01   5.73111e-03        488    u    2    1
  3  -5.23667e-01   1.90703e-02        456    u    2    0
  4  -5.23670e-01   1.90685e-02        456    u    2    0
------------------------------------------------------------
 Total error:                            2.83367e-02    0
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       7.619212089928772e+01     3.51480e-01    1
 N-E          -1.990304522149214e+02    -5.06365e-01    1
 Coulomb       4.663204498156729e+01     1.48732e-01    1
 Exchange     -2.231975936206787e+00    -6.16493e-03    1
 X-C          -7.018784412244681e+00    -1.98337e-02    1
------------------------------------------------------------
 Electronic   -8.545704668251783e+01    -3.21522e-02    1
 Nuclear       9.039235444235468e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.641781123828235e+01    -3.21522e-02    1
============================================================


################ Wall time: 8.77182e+00 sec ################




####################### SCF cycle  4 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
 Nuclear energy                       9.039235444235468e+00
 Electronic energy                   -8.544852693431930e+01
------------------------------------------------------------
 Total energy                        -7.640929149008383e+01
------------------------------------------------------------
                 Wall time: 2.81517e-01 sec 
============================================================


============================================================
              Initializing Helmholtz Operators
------------------------------------------------------------
 Creating operator with lambda                 -1.91095e+01
 Creating operator with lambda                 -7.09513e-01
 Re-using operator with lambda                 -7.09513e-01
 Creating operator with lambda                 -5.23667e-01
 Re-using operator with lambda                 -5.23667e-01
------------------------------------------------------------
                 Wall time: 2.94490e-01 sec 
============================================================


============================================================
                Applying Helmholtz Operators
------------------------------------------------------------
 Orb  RealNorm     ImagNorm      nNodes     Error   Timing   
------------------------------------------------------------
  0  9.99883e-01  0.00000e+00   456     0  1.2e-04  5.1e-01
  1  1.00044e+00  0.00000e+00   360     0  4.4e-04  4.5e-01
  2  1.00041e+00  0.00000e+00   360     0  4.1e-04  4.5e-01
  3  9.98247e-01  0.00000e+00   392     0  1.8e-03  4.7e-01
  4  9.98250e-01  0.00000e+00   392     0  1.8e-03  4.7e-01
------------------------------------------------------------
                 Wall time: 2.35322e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             8.07000e-03
 Setup linear system                            1.71745e-01
 Solve linear system                            1.00000e-05
 Expand solution                                9.94140e-02
------------------------------------------------------------
                 Wall time: 2.79343e-01 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     872      5.05050e-02
 Coulomb density                       616      9.73650e-02
 Coulomb potential                     136      1.58912e-01
 Hartree-Fock exchange                 616      3.64690e+00
 XC density                            616      1.02887e-01
 XC density gradient                  1848      1.37320e-02
 XC preprocess xcfun                  1232      6.23490e-02
 XC evaluate xcfun                    1848      1.47614e-01
 XC energy                             616      4.20000e-05
 XC potential                          616      8.29240e-02
------------------------------------------------------------
                 Wall time: 4.36909e+00 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0  -1.90999e+01   1.80889e-03        552    u    2    1
  1  -7.10591e-01   6.61736e-03        488    u    2    1
  2  -7.10641e-01   6.61963e-03        488    u    2    1
  3  -5.20351e-01   5.19330e-03        456    u    2    1
  4  -5.20354e-01   5.19387e-03        456    u    2    1
------------------------------------------------------------
 Total error:                            1.20344e-02    1
============================================================


============================================================
                    Energy                 Update      Done 
------------------------------------------------------------
 Kinetic       7.630873843521852e+01     1.16618e-01    1
 N-E          -1.990545472072562e+02    -2.40950e-02    1
 Coulomb       4.653966018261868e+01    -9.23848e-02    1
 Exchange     -2.229377365590050e+00     2.59857e-03    1
 X-C          -7.013000979310207e+00     5.78343e-03    1
------------------------------------------------------------
 Electronic   -8.544852693431930e+01     8.51975e-03    1
 Nuclear       9.039235444235468e+00     0.00000e+00    1
------------------------------------------------------------
 Total        -7.640929149008383e+01     8.51975e-03    1
============================================================


################ Wall time: 8.90013e+00 sec ################


============================================================
                    Localizing orbitals
------------------------------------------------------------
 Computing Foster-Boys matrix                   9.31044e-01
 Converged after iteration                                2
 Rotating orbitals                              3.18750e-02
------------------------------------------------------------
                 Wall time: 9.62990e-01 sec 
============================================================


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    9.19958e-02    -7.633861620975058e+01   -7.63386e+01
  2    3.40329e-02    -7.638565905209957e+01   -4.70428e-02
  3    2.83367e-02    -7.641781123828235e+01   -3.21522e-02
  4    1.20344e-02    -7.640929149008383e+01    8.51975e-03
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     872      5.01830e-02
 Coulomb density                       616      9.77460e-02
 Coulomb potential                     136      1.60038e-01
 Hartree-Fock exchange                 616      3.61623e+00
 XC density                            616      9.67900e-02
 XC density gradient                  1848      1.35510e-02
 XC preprocess xcfun                  1232      6.15700e-02
 XC evaluate xcfun                    1848      1.47308e-01
 XC energy                             616      3.70000e-05
 XC potential                          616      7.92290e-02
------------------------------------------------------------
                 Wall time: 4.32790e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 2.78743e-01 sec 
============================================================


============================================================
                 Calculating dipole moment
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 2.96951e-01 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-1.91010e+01 4.99730e-01 4.99671e-01 -9.60644e-01 -9.60646e-01
5.00898e-01 -7.11550e-01 -1.62002e-01 1.78935e-01 1.78935e-01
5.00838e-01 -1.62001e-01 -7.11529e-01 1.78939e-01 1.78938e-01
-9.60900e-01 1.79017e-01 1.79022e-01 -5.18875e-01 -1.72713e-01
-9.60901e-01 1.79017e-01 1.79020e-01 -1.72713e-01 -5.18874e-01
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    u    2                      -1.923000498354142e+01
    1    u    2                      -1.048120724361395e+00
    2    u    2                      -5.495381111463084e-01
    3    u    2                      -3.879887356382435e-01
    4    u    2                      -3.461615754724336e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     O           0.00000e+00   0.00000e+00  -1.25000e-01
  2     H          -1.43750e+00   0.00000e+00   1.02500e+00
  3     H           1.43750e+00   0.00000e+00   1.02500e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   3.68310e-03
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -4.312362826031961e+01
 Kinetic energy:                      7.638281413586279e+01
 E-N energy:                         -1.991777377986874e+02
 Coulomb energy:                      4.658316805784973e+01
 Exchange energy:                    -2.230956155407826e+00
 X-C energy:                         -7.018105582766777e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy                   -8.546081734314950e+01
 Nuclear energy                       9.039235444235468e+00
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.642158189891403e+01
                    (kJ/mol)         -2.006448342353977e+05
                    (kcal/mol)       -4.795526627752755e+04
                    (eV)             -2.079537091181460e+03
                                                            
============================================================
                                                            

                                                            
============================================================
                         Dipole moment                      
------------------------------------------------------------
                                                            
 Length of vector:   (au)                  7.6097371415e-01
                     (Debye)               1.9342020699e+00
                                                            
------------------------------------------------------------
                                                            
   9.3856969261e-05   -1.2573816734e-06   -7.6097370836e-01
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 5.166951e+01                ***
***                                                      ***
************************************************************


